code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -51,6 +52,8 @@
 	origin_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 5, TECH_POWER = 3)
 	slot_flags = SLOT_BELT
 	force = 8 //looks heavier than a pistol

```

Comment:
It does appear to be heavier than a pistol according to this comment--but it's still a pistol. Just a fat pistol.